src/main/java/org/prebid/cache/repository/aerospike/AerospikePropertyConfiguration.java

```diff
@@ -27,6 +28,7 @@
 @AllArgsConstructor
 @Configuration
 @EnableConfigurationProperties
+@ConditionalOnProperty(prefix = "spring.aerospike", name = {"endpoint"}, havingValue = "true")
```

Comment:
There is no need for a new `endpoint` property since there is a couple of other required properties that could be used instead, for example `host`, same for redis

Reply:
for now in application.yml we have properties for both.

Reply:
aerospike and redis

Reply:
I believe it is something that has to change. It's only in `local` profile so either aerospike or redis configuration could be left and another removed.